gce/image/build_image.sh

```diff
@@ -163,6 +162,7 @@ else
 
 fi
 
+SCYLLA_IMAGE_NAME=$(echo $SCYLLA_VERSION | sed "s/\./-/g")
```

Comment:
are you sure you need the sed part? `clean_resource_name` will normalize the name..

Reply:
and better to put $(...) in quotes

Reply:
Let me verify it

Reply:
@bentsi you were right. it's not needed. removed